minimal/compose/nginx/production/nginx.conf

```diff
@@ -36,7 +36,15 @@ http {
         listen 80;
         listen [::]:80;
         server_name {{ DOMAIN }};
-        return 301 https://$server_name$request_uri;
+
+        # Certbot always performs validations over HTTP
+        location ^~ /.well-known  {
```

Comment:
No, we need `.well-known` to be accessible on port 80 so that location block needs to be there. We don't however need it on 443 since it's never accessed there.